src/components/common/InputTextarea/InputTextarea.jsx

```diff
@@ -0,0 +1,5 @@
+import styles from './InputTextarea.module.css';
+
+export default function InputTextArea(placeholder) {
```

Comment:
```suggestion
export default function InputTextArea({placeholder}) {
```
이 부분 props 로 받는 부분이면 중괄호로 감싸 주세요!

Reply:
빠른 피드백 감사드립니다 바로 수정했습니다!